src/core/qgsvectorlayerrenderer.cpp

```diff
@@ -35,6 +35,7 @@
 #include "qgsexception.h"
 #include "qgslogger.h"
 #include "qgssettings.h"
+#include "qgsproject.h"
```

Comment:
May as well revert these changes to qgsvectorlayerrender.cpp/.h - they look like leftovers from an earlier version of this pr